pg_metadata/locator.py

```diff
@@ -72,10 +72,12 @@ def fetch_result_single_database(self, search: str, connection_name: str):
             )
 
         # Search items from pgmetadata.dataset
-        sql = " SELECT concat(title, ' (', table_name, '.', schema_name, ')') AS displayString,"
-        sql += " schema_name, table_name, geometry_type, title"
-        sql += " FROM pgmetadata.dataset"
-        sql += " WHERE concat(title, ' ', abstract, ' ', table_name) ILIKE '%{}%'".format(search)
+        sql = " SELECT concat(d.title, ' (', d.v_valid_dataset, '.', d.schema_name, ')') AS displayString,"
+        sql += " d.schema_name, d.table_name, d.geometry_type, title"
+        sql += " FROM pgmetadata.dataset d"
+        sql += " INNER JOIN pgmetadata.v_validate_dataset v ON concat(v.v_valid_dataset, '.', v.schema_name)"
```

Comment:
I just tried the PR, because I have the demo soon : 
```
2020-11-17T12:43:38     CRITICAL    LocatorFilter: SQL error: SELECT concat(d.title, ' (', d.v_valid_dataset, '.', d.schema_name, ')') AS displayString, d.schema_name, d.table_name, d.geometry_type, title FROM pgmetadata.dataset d INNER JOIN pgmetadata.v_validate_dataset v ON concat(v.v_valid_dataset, '.', v.schema_name) = concat(d.table_name, '.', d.schema_name) WHERE concat(d.title, ' ', d.abstract, ' ', d.table_name) ILIKE '%comm%' LIMIT 100 returned 0 [ERROR: relation "pgmetadata.v_validate_dataset" does not exist
             LINE 1: ..._type, title FROM pgmetadata.dataset d INNER JOIN pgmetadata...
              ^
             ]
```

`INNER JOIN pgmetadata.v_validate_dataset`

The locator is currently broken, the view is called `v_valid_dataset`, not `validate`. Fixing it.